operatorcourier/nest.py

```diff
@@ -62,7 +64,8 @@ def nest_bundles(yaml_files, registry_dir, temp_registry_dir):
             crd_name = csv_crd["name"]
             if crd_name in crds:
                 crd = crds[crd_name]
-                with open('%s/%s.crd.yaml' % (csv_folder, crd_name), 'w') as outfile:
+                crdfile_name = os.path.join('%s', '%s.crd.yaml') % (csv_folder, crd_name)
```

Comment:
```suggestion
                crdfile_name = os.path.join(csv_folder, '%s.crd.yaml') % crd_name
```

Reply:
nope, this is correct
```python
os.path.join(csv_folder, '%s.crd.yaml' % crd_name')
```

Reply:
resp. both are correct, but way I mentioned seems more clear to me and less error prone

Reply:
@MartinBasti Thanks for the suggestion!